### PR TITLE
spec for teamdctl_config_dump

### PR DESCRIPTION
**Pull request by rasrivas-redhat**

### CODE REVIEW

File: uploader.json [hunk 1]
```diff
@@ -7413,6 +7419,16 @@
                 }
             ]
         },
+        "teamdctl_config_dump": {
```
Comment (xiangce) [inline uploader.json]: The content in `specs` is deprecated, so this item is not needed.

Comment (rasrivas-redhat) [reply uploader.json]: @xiangce - Acknowledge, updated the code as per your suggestion

File: uploader.json [hunk 2]
```diff
@@ -882,6 +882,12 @@
             "pattern": [],
             "symbolic_name": "systool_b_scsi_v"
         },
+        {
+            "command": "/usr/bin/teamdctl  config dump",
```
Comment (bfahr) [inline uploader.json]: @rasrivas-redhat will this command work if the iface argument is appended to the end of this command?  That is the way pre_commands work, they are appended to the end of the command string.

Comment (rasrivas-redhat) [reply uploader.json]: @bfahr - Apology for the late response as I was on PTO on 22 March.

1. This command will not work if iface argument is appended to the end of this command
2. We already have "**teamdctl_state_dump**" command, which works in the same way, accordingly I have updated the code for "**teamdctl_config_dump**", please let me know if any changes are required.

**Comment by rasrivas-redhat** [comment]
Closing this PR, as per discussion with @xiangce , there is no need for this spec

**Pull request closed without merging**